drop_service/views.py

```diff
@@ -37,7 +40,12 @@ def _get_drops(self, drop_id):
         try:
             have_since, since = self.get_if_modified_since()
         except ValueError as value_error:
-            return error(str(value_error)), None
+            logger.warning('Could not parse modified-since header pack: %s', value_error)
+            have_since, since = False, None
+
+        if have_since and not since:
```

Comment:
This block was on the wrong abstraction layer. I moved it to get_if_modified_since, which also makes it much more clear where this came from.